src/reference/proxying.md

```diff
@@ -0,0 +1,53 @@
+# Proxying
+
+## Listeners
+
+By default Caido will listen on `127.0.0.1:8080`, it uses a traffic splitting algorithm to determine if a request is for the UI/[GraphQL API](/concepts/internals/graphql) or if it a normal user request that needs to be forwarded.
+
+To change the main listener, use the CLI argument `-l <IP:PORT>` as outlined in the [guide](http://localhost:5173/guides/listening_address.html).
+
+To add other listeners you have to use the follow arguments, you can repeat them as many times as you need:
+
+- `--ui-listen <IP:PORT>`: This will listen on the given address ONLY for the UI/GraphQL API
+- `--proxy-listen <IP:PORT>`: This will listen on the given address ONLY for the proxy
+
+::: info
+Currently we do not offer the multi-listeners configuration in the Desktop application.
+:::
+
```

Comment:
We should move the additional info about `--ui-listen` and `--proxy-listen` to the `guides/listening_address.html` section